conda/core/package_cache_data.py

```diff
@@ -729,10 +824,67 @@ def __eq__(self, other):
         return hash(self) == hash(other)
 
 
+# called from ProgressiveFetchExtract.execute()
+def do_cache_action(prec, cache_axn, progress_bar, download_total=1.0):
+    # dummy action to simplify code
+    if not cache_axn:
```

Comment:
cache_action? Not sure what `axn` means. Let's remove that ambiguity.

Reply:
Realizing now that it's used else where in the code, but let's not follow a bad convention!